### PR TITLE
Add bin/nanopub-java.bat and include it in installation for windows

### PR DESCRIPTION
**Pull request by svenvanderburg**
Makes package work on Windows
* Added nanopub-java.bat file to run the `nanopub-java` on windows
* Several small fixes in python code to make compatible with windows
* Tested extensively locally on a windows machine, all tests pass, but I could not get it to work in a github action. I created #118 to fix it later.

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -7,31 +7,39 @@ on: [push, pull_request]
 
 jobs:
   build:
-
-    runs-on: ubuntu-latest
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [windows-latest]
```
Comment (svenvanderburg) [inline .github/workflows/build.yml:13]: TODO: This should also include ubuntu-latest

File: nanopub/setup_nanopub_profile.py [hunk 2]
```diff
@@ -39,12 +39,11 @@ def validate_orcid_id(ctx, orcid_id: str):
                     '(by default HOMEDIR/.nanopub/). '
                     'The profile will also be published to the nanopub servers.')
 @click.option('--keypair', nargs=2, type=Path,
-              prompt=f'If the public and private key you would like to use are not '
-                     f'in {USER_CONFIG_DIR}, provide them here. '
-                     f'If they are in this directory or you wish to generate new keys, '
-                     f'leave empty.',
               help='Your RSA public and private keys with which your nanopubs will be signed',
               default=None)
+@click.option('--newkeys', type=bool, is_flag=True, default=False,
```
Comment (svenvanderburg) [inline nanopub/setup_nanopub_profile.py:44]: I added this so we can just call `setup_nanopub_profile` fully non-interactively from the command line (which is much nicer for github action).

File: bin/nanopub-java.bat [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+@rem Get command-line arguments, handling Windows variants
+
+if not "%OS%" == "Windows_NT" goto win9xME_args
+
+:win9xME_args
+@rem Slurp the command line arguments.
+set CMD_LINE_ARGS=
+set _SKIP=2
+
+:win9xME_args_slurp
```
Comment (raar1) [inline bin/nanopub-java.bat:55]: This is so... beautiful...

File: nanopub/setup_nanopub_profile.py (hunk 2 above)
Comment (raar1) [reply nanopub/setup_nanopub_profile.py:44]: Yes, this is a much needed option.

**Review by raar1 — APPROVED** [review]
Well done for persevering with this nightmare. As discussed before, getting the tests to pass on windows CI is proving very difficult, but it seems to work locally and windows support is not a priority unless/until we have users on that OS, so let's leave it for now and return once we have more time.

**Comment by raar1** [comment]
We should disable the windows CI tests for now, before merging.

**Pull request merged**